docs/content/getting-started/_index.md

```diff
@@ -9,49 +9,82 @@ menu = "start"
+Use [NPM](https://www.npmjs.com/) (Node Package Manager) to install individual PatternFly Elements.
 
 ```bash
 npm install --save @patternfly/pfe-card
+npm install --save @patternfly/pfe-cta
 ```
 
-Or, utilize the Red Hat asset library! When it exists! 
-
 
+## 2. Include PatternFly Elements JavaScript
 
-## 2. Include PatternFly Elements on a page
-
-You have some options:
+There are a few options:
 
 1. Load JavaScript modules via script tag: `script type="module"`
-	1.  Downloads all of the dependencies on its own
-	2.  [Only supported in modern browsers](https://caniuse.com/#search=module)
+	1.  Downloads all of the dependencies on its own.
+	2.  [Only supported in modern browsers](https://caniuse.com/#search=module).
 2. Include the PatternFly Element and its dependencies on the page(s) or within the app.
 
 	```html
 	import '@patternfly/pfe-card/pfe-card.js';
 	import '@patternfly/pfe-cta/pfe-cta.js';
 	```
 
-3. Use [require.js](https://requirejs.org/) JavaScript file and module loader
+3. Use [require.js](https://requirejs.org/) JavaScript file and module loader.
 	- Learn more about [Polyfills](/getting-started/polyfills)
-3. Load individual PatternFly Element scripts, but bundle the polyfills with the base `pfelement.js` file
-	1.  All elements are based off of pfelement.js so including the polyfills with this one file would mean you only need to include the pfelement.js file before you include anything else
-4. Bundle all of the scripts together into one
-5. TBD Idea: polyfill.io > webservice (build something similar internal to Red Hat - component service)
-	1.  pipe needed polyfills for website, checks headers, runtime
-	2.  delivers dynamic JS
-	3.  If I want pfe-card - it gets the pfe-card.js + required other files (serves up js bundle)
+3. Load individual PatternFly Element scripts, but bundle the polyfills with the base `pfelement.js` file.
+	1.  All elements are based off of pfelement.js so including the polyfills with this one file would mean you only need to include the pfelement.js file before you include anything else.
+4. Bundle all of the scripts together into one rollup, and include that.
+
 
 
 
 ## 3. Use PatternFly Elements markup
 
-You can use PatternFly Elements with other standard HTML markup in your app or page:
+Different components have different intended uses. We tend to thing of them in 3 distinct groups:
+
+- Content, such as a call-to-action: `pfe-cta`
+- Containers, such as a card: `pfe-card`
+- Combos, such as the pattern which captures sets of content and renders it as an accordion or tab set: `pfe-content-set`
+
+
+### Content Components
+
+The beauty of web components is that they have much of the styling built-into the tag itself. Start with the tags first.
+
+```
+<body>
+    <pfe-cta>
+        <a href="#">Learn more</a>
+    </pfe-cta>
+</body>
+```
+
+
+
+### Container components (see also container notes below)
+
+```
+<body>
+    <pfe-card pfe-color="darkest">
+        Hello world.
```

Comment:
Maybe wrap this in a p-tag since unwrapped text is problematic in components and it might not be good to show many demos doing that?

Reply:
thx, updated